### DISCUSSION TITLE
Org Metrics Dashboard v2 shipped!

### DISCUSSION THREAD

**Discussion by ahpook** [community/OSPO#7] [Dashboard]
On Oct 28 we shipped an all-new version of the org metrics dashboard! Some highlights:

* **Updated UI.** The look and feel are now more “GitHub-esque” through the use of our open source design system, [Primer](http://github.com/primer).
Advanced filtering. You can now filter the list of repositories using a familiar text-based syntax, akin to PR and Issue searches. For example, `has:readme -has:coc` will display repos with a README but no Code of Conduct file.
* **Improved sorting.** Selecting the header columns will now sort the list of displayed repos in alternating directions.
* **New visualizations.** Selecting an individual repo name will display new flyout visualizations for contribution and issue time-series data.
* **Documentation.** Early-access docs are now available [here](https://docs.github.com/en/early-access/insights/organization-metrics-dashboard)

Please give it a try and post a comment on this discussion if you've got feedback about any of the new goodies!

**Comment by syrusakbary** [comment]
I'd love to try OSPO for both the [Wasmer](https://github.com/wasmerio) and [GraphQL Python](https://github.com/graphql-python/) orgs!

**Reply by ahpook** [reply]
Awesome @syrusakbary - I've added these to the queue for enablement; I will drop you an email when they're up and running.

**Comment by caabernathy** [comment]
I was wondering where to report a potential issue. It would seem that if a contribution's author and committer are different, then the author isn't counted as the contributor in the org-level representation of contribution insights.  This differs from the Insights on the repo level which properly credits the author.

**Comment by lelia** [comment]
Hey @ahpook we'd love to preview this in our [/wayfair](https://github.com/wayfair) and [/wayfair-incubator](https://github.com/wayfair-incubator) orgs! 

We've also filed an issue [here](https://github.com/community/OSPO/issues/14) requesting access, in case that's the better place to do so 🙂

**Comment by justinabrahms** [comment]
hi. The primer link above is a 404.

I noticed in my "Average time to resolve issue by days" graph, the y-axis isn't 0 based which made it seem not nearly as bad as it actually is. 😅 Would love it if those were 0'd out.

<img width="538" alt="image" src="https://user-images.githubusercontent.com/3853/205391495-dbca844d-bd5a-44d9-bd5a-72f338c36c52.png">

**Reply by ahpook** [reply]
Updated the link - Primer got promoted to its own org and website: https://primer.style

**Reply by ahpook** [reply]
Good point about the Y axis - we are looking at improving the visualizations over the next couple of iterations and will take that into account!

**Comment by justinabrahms** [comment]
Another one. How is the license determined? Is this something that can be explicitly set? [This project](https://github.com/eBay/DASTProxy/blob/master/LICENSE), as an example, is MIT licensed but does discuss the other licenses of its dependencies.

<img width="360" alt="image" src="https://user-images.githubusercontent.com/3853/205393887-99ea9288-b253-4014-83ed-54c67236a390.png">

**Reply by lelia** [reply]
Had this same question!

**Reply by binford2k** [reply]
tldr; include the exact copy of the license you want to use as the `LICENSE` file.

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#detecting-a-license

**Comment by justinabrahms** [comment]
Another one for you. The dashboard's "no contribution since" calculation doesn't appear to handle cases where the repository is new and hasn't yet gotten a contribution.
<img width="743" alt="image" src="https://user-images.githubusercontent.com/3853/205395075-7e64a553-6242-4872-a9b7-019b33e7ed48.png">

<img width="977" alt="image" src="https://user-images.githubusercontent.com/3853/205394806-26db47c6-9c9c-413a-9dcd-29260940ea5d.png">

**Comment by bastelfreak** [comment]
Hi @ahpook. awesome work! I use the insights for the Vox Pupuli org and I think it's quite helpful. I think the limit of 15 repos per page is a bit small for bigger screens. Maybe that can be made configureable?


Edit: Some repos are listed with a missing license. We added them, but the repo overview isn't updated. How often is it updated/is there a way for us to trigger an update?

**Reply by binford2k** [reply]
I noticed the same thing with one of ours (puppet-strings). It got the updated license in roughly 24 hours and is now accurate.
